Week3/More-JS-7-8-9-10-11.js

```diff
@@ -0,0 +1,51 @@
+
+//7 Make a list of vehicles, you can add "motorbike", "caravan", "bike", or more.
+let listOfVehicles = ["motorbike","caravan","bike"];
+
+//8 How do you get the third element from that list?
+console.log(listOfVehicles[2]);
+
+
+
+
+/*9 Change the function vehicle to use the list of question 7.
+ So that vehicle("green", 3, 1) prints "a green new bike".*/
+function Vehicles(color,code,age){
```

Comment:
This is almost correct, because right now I cannot print "a green new car". Can you change the code to allow that also (and also "a red new motorbike")